### DISCUSSION TITLE
Launch command strange behaviour

### DISCUSSION THREAD

**Discussion by Venikeee1** [kovidgoyal/kitty#7719] [General]
I have mapping like this `map f1 launch --title=lazygit --cwd=current --type=overlay lazygit`. So when I hit f1 `lazygit` is opened in overlay type. Problem with it is that I lose all highlights in diff section. Also it doesn't recognise npm in git hooks. I know that you can say that it is something with `lazygit` but when I open it via splitting the window and typing `lazygit` in it everything works fine. Could it be something related to kitty child process?
![2024-08-1223-41-42-ezgif com-optimize](https://github.com/user-attachments/assets/53cd9288-c7aa-442c-a662-8785a8f5e459)

**Comment by kovidgoyal** [comment]
It will be because lazygit is relying on something in your shell's
environment probably PATH. You need to do something like

launch zsh -il -c "exec lazygit"

replace zsh with whatever your shell is. That will cause the shell to
read its rc files and setup the environment before running lazygit.

**Reply by Venikeee1** [reply]
That worked. Many thanks
